quarkus-service-registry/README.adoc

```diff
@@ -0,0 +1,249 @@
+* You copied the *Token endpoint URL* from the same list of endpoints to be used for the OAuth based athentication method used in this example.
+* You have generated a service account from the Service Accounts section of {product} and stored in a secure place.
+
+endif::[]
+
+.Procedure
+. On the command line, set the Kafka and {registry} instance URLs, token endpoint URL, and client credentials as environment variables to be used by Quarkus or other applications. Replace the values with your own server and credential information:
++
+--
+ifdef::qs[]
+* The `<bootstrap_server>` is the bootstrap server endpoint for your Kafka instance. 
+* The `<registry_uri>` is the CORE API endpoint for your {registry} instance. 
+* The `<client_id>` and `<client_secret>` are the generated credentials for your service account.
+* The `<oauth_token_endpoint_uri>` is the SASL/OAUTHBEARER token endpoint. 
+endif::[]
+
+.Setting environment variables for server and credentials
+[source,subs="+quotes"]
+----
+$ export BOOTSTRAP_SERVER=__<bootstrap_server>__
+$ export REGISTRY_URL=__<registry_uri>__
+$ export CLIENT_ID=__<client_id>__
+$ export CLIENT_SECRET=__<client_secret>__
+$ export OAUTH_TOKEN_ENDPOINT_URI=__<oauth_token_endpoint_uri>__
+----
+--
+. In the Quarkus example application, review the `/src/main/resources/application.properties` files in the `consumer` and `producer` sub-folders to understand how the environment variables you set in the previous step are used. This example uses the `dev` configuration profile in the `application.properties` files.
+
+ifdef::qs[]
+.Verification
+* Did you set the required environment variables for accessing your services?
+endif::[]
+
+[id="proc-create-prices-topic-registry_{context}"]
+== Creating the requests and quotes Kafka topics in {product}
+
+[role="_abstract"]
+For this quick start, the Kafka topics that the Quarkus example application references are called `requests` and `quotes`. You need to create these topics in {product} so that the Quarkus application can interact with it.
+
+.Prerequisites
+* You've created a Kafka instance in {product} and the instance is in *Ready* state.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to add a topic to.
+. Select the *Topics* tab, click *Create topic*, and follow the guided steps to define the topic details. Click *Next* to complete each step and click *Finish* to complete the setup.
++
+[.screencapture]
+.Guided steps to define topic details
+image::sak-create-topic.png[Image of wizard to create a topic]
+
+* *Topic name*: Enter `requests` as the topic name.
```

Comment:
No more requests topic. The situation right now is that there are two applications, a producer and a consumer. The producer produces quotes every 5 seconds to the quotes topic and the consumer consumes from that topic.